generators/server/templates/pom.xml.ejs

```diff
@@ -128,6 +128,7 @@
         <%_ } _%>
         <properties-maven-plugin.version>1.0.0</properties-maven-plugin.version>
         <sonar-maven-plugin.version>3.6.0.1398</sonar-maven-plugin.version>
+        <asciidoctor-maven-plugin.version>1.5.6</asciidoctor-maven-plugin.version>
```

Comment:
@sahbi-ktifa Can you please use version 1.6.0 so it is compatible with java 8+

Reply:
Or we can target to version v2.0.0-RC.1?

Reply:
I followed asciidoctor maven/gradle plugin recommendations, but yes, we should definitely stick to the latest version!